reference/resources/browser_info.yaml

```
@@ -0,0 +1,68 @@
+---
+title: Browser info
+type: object
+
+required:
+  - java_enabled
+  - javascript_enabled
+  - language
+  - color_depth
+  - screen_height
+  - screen_width
+  - time_zone_offset
+  - user_device
+
+properties:
+  java_enabled:
+    type: boolean
+    description: Indicates whether the client browser supports Java.
+    example: true
+
+  javascript_enabled:
+    type: boolean
+    description: Indicates whether the client browser supports JavaScript.
+    example: true
+
+  language:
+    type: string
+    description: |-
+      The preferred language of the buyer,
+      usually the language of the browser UI.
+    example: "en-GB"
+
+  color_depth:
+    type: string
+    description: The color depth of the screen.
+    example: "32"
+    enum:
+      - "1"
+      - "4"
+      - "8"
+      - "15"
+      - "16"
+      - "24"
+      - "32"
+      - "48"
+
+  screen_height:
+    type: number
+    description: The height of the screen in pixels.
+    example: 1080
+
+  screen_width:
+    type: number
+    description: The width of the screen in pixels.
+    example: 1920
+
+  time_zone_offset:
+    type: number
+    description: Time-zone offset in minutes between UTC and buyer location.
+    example: 60
+
+  user_device:
+    type: string
+    description: The platform that is being used to access the website.
+    example: "DESKTOP"
+    enum:
+      - "DESKTOP"
+      - "MOBILE"
```

```suggestion
    example: "desktop"
    enum:
      - "desktop"
      - "mobile"
```

We generally use lower case enums in our API.